app/src/layers/components/LayerItem.jsx

```diff
@@ -124,14 +124,23 @@ class LayerItem extends Component {
           </li>}
           <li
             className={LayerItemStyles.itemOptionItem}
+            onClick={() => this.onClickInfo()}
+          >
+            <InfoIcon
+              className={IconStyles.infoIcon}
+            />
+          </li>
+
+          {/* <li
```

Comment:
I suggest removing the commented code in a separated commit, with a message explaining why it was removed.
If you'd rather keep it here, please add an explanation along with the commented code.